pkg/migrations/trigger.go

```diff
@@ -41,6 +41,11 @@ type createTriggerAction struct {
 	cfg  triggerConfig
 }
 
+type dropTriggerAction struct {
```

Comment:
Shouldn't this and the related code go with the other actions in `dbactions.go`?

Reply:
Also, this appears to duplicate the existing `DropFunctionAction`